indico/modules/events/persons/templates/management/_person_list_row.html

```diff
@@ -54,9 +54,17 @@
         {% if person_data.roles.no_account %}
             <span class="i-label warning icon-warning"
                    title="{% trans email=person.email %}This user seems to have no Indico
-                   account. If they do have an account, they should add {{ email }} to their profile
+                   account. If they do have an account, they should add {{ email }} to their profile{%- endtrans %}">
+                   {% trans %}No Indico account{% endtrans %}
+            </span>
+        {% endif %}
+    </td>
+    <td class="i-table">
+        {% if person_data.person.invited_dt %}
```

Comment:
in line 2 we assign `person` so you can replace `person_data.person` with just `person` here and two lines below